gateway/ln-gateway/src/lib.rs

```diff
@@ -252,6 +252,8 @@ impl Gateway {
             GatewayError::Other(anyhow::anyhow!("Invalid federation member string {}", e))
         })?;
 
+        let _ = self.select_actor(connect.id.clone()).await?;
```

Comment:
Pretty sure this will prevent us from connecting any new federations! Have you tested this?
Can you verbally describe the behavior you are expecting and see if this check matches your expectations?

Reply:
ah youre right, i was kinda sleepy when i did this. anyway, dont think it deserves this much discussion, feel free to add it in some other patch.